### DISCUSSION TITLE
Showing an image at the top of an extruded polygon

### DISCUSSION THREAD

**Discussion by renjithraj2005** [maplibre/maplibre-gl-js#4190] [General]
Can we show an image similar to the one below?
<img width="384" alt="Screenshot 2024-05-29 at 2 15 05 AM" src="https://github.com/maplibre/maplibre-gl-js/assets/895314/77d7709a-be16-4e4f-9182-8859443c9d60">

**Comment by HarelM** [comment]
I'm not aware of an ability to do that, but you can try the maplibre slack channel, maybe someone over there has a solution.
